services/cards-publication/src/main/java/org/opfab/cards/publication/model/LightCard.java

```diff
@@ -35,6 +36,8 @@ public record LightCard(
         String titleTranslated,
         String summaryTranslated,
         List<TimeSpan> timeSpans,
+        @JsonIgnore
```

Comment:
I do not understand why it shall be ignore

Reply:
Because it is not needed on frontend side